src/js/letters/actions/letters.js

```diff
@@ -189,3 +191,42 @@ export function updateAddress(address) {
     address
   };
 }
+
+export function saveAddressPending() {
+  return {
+    type: SAVE_ADDRESS_PENDING
+  };
+}
+
+export function saveAddressSuccess(address) {
+  return {
+    type: SAVE_ADDRESS_SUCCESS,
+    address
+  };
+}
+
+export function saveAddressFailure(address) {
+  return {
+    type: SAVE_ADDRESS_FAILURE,
+    address
+  };
+}
+
+export function saveAddress(address) {
+  const settings = {
+    method: 'PATCH',  // TODO: decide whether to use PATCH or PUT here; check with Alastair
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(address)
+  };
+  return (dispatch) => {
+    // TODO: Show a spinner or some kind of indication we're waiting on this to return
```

Comment:
Good idea: can you write a ticket for it?

Reply:
Already done 🙂 
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4859